docker/router.dockerfile

```diff
@@ -31,7 +31,7 @@ RUN touch ./src/main.rs
 RUN cargo build --release
 
 # Runtime
-FROM debian:bullseye-slim as runtime
+FROM debian:latest as runtime
```

Comment:
@kamilkisiela can we use the explicit version of `latest`?

Reply:
yes